### PR TITLE
Create docs for the VDC dashboard package

### PR DESCRIPTION
**Pull request by dmahmouali**
### Changes

- Create docs for the VDC dashboard package
- And for Adding new solutions chatflow its updated in this PR https://github.com/threefoldtech/js-sdk/pull/2839
### Related Issues

-  https://github.com/threefoldtech/js-sdk/issues/1924
### Checklist

- [ ] [Pre-commit hook is installed](https://github.com/threefoldtech/js-ng#pre-commit) to do formatting checks before committing code...etc
- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings

### CODE REVIEW

File: docs/vdc/vdc_dashboard.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  - `Add new worker node`
+  - `Delete existing worker node`
+  - `Get all storage nodes`
+  - `Add new storage node`
+  - `Delete storage node`
+  - `Get vdc prepaid wallet info`
+  - `Get all used pools`
+  - `Get alerts`
+### The Package Frontend
+- The frontend built with [Vue js](https://vuejs.org/) and use [vuetify js](https://vuetifyjs.com/) some components.
```
Comment (ranatrk) [inline docs/vdc/vdc_dashboard.md:49]:
```suggestion
- The frontend is built with [Vue js](https://vuejs.org/) using some components from [vuetify js](https://vuetifyjs.com/).
```

File: docs/vdc/vdc_dashboard.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+- The VDC Marketplace home page:
+  - The user can deploy, list and delete his/her solutions.
```
Comment (ranatrk) [inline docs/vdc/vdc_dashboard.md:52]:
```suggestion
  - The user can deploy, list, and delete their solutions.
```

File: docs/vdc/vdc_dashboard.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+![vdc_marketplace](./images/vdc_marketplace.png)
+
+- The VDC setting home page:
+  - The default tab represent the `Compute Nodes` for the VDC Kubernetes cluster, The user can Add or remove the nodes.
```
Comment (ranatrk) [inline docs/vdc/vdc_dashboard.md:56]:
```suggestion
  - The default tab represents the `Compute Nodes` for the VDC Kubernetes cluster where the user can add or remove nodes from their VDC.
```

File: docs/vdc/vdc_dashboard.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  ![vdc_setting](./images/vdc_setting.png)
+  - In the `Storage Nodes` tab the user can:
+    - Add/Delete storage node.
```
Comment (ranatrk) [inline docs/vdc/vdc_dashboard.md:59]:
```suggestion
    - Add/Delete storage nodes.
```

File: docs/vdc/vdc_dashboard.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    - Enable quantum storage.
+    - Download [Z-stor](https://github.com/threefoldtech/0-stor_v2) configuration.
+    - Download ZDBs information.
+
+    ![vdc_storage_nodes](./images/vdc_storage_nodes.png)
+
+  - In Wallet Information tab there is the prepaid wallet information.
```
Comment (ranatrk) [inline docs/vdc/vdc_dashboard.md:66]:
```suggestion
  - `Wallet Information tab`: consists of the prepaid wallet information.
```

File: docs/vdc/vdc_dashboard.md (hunk 3 above)
Comment (ranatrk) [inline docs/vdc/vdc_dashboard.md:56]:
```suggestion
  - `Compute Nodes tab` is the default tab for the VDC Kubernetes cluster, where the user can add or remove their VDC's nodes.
```

File: docs/vdc/vdc_dashboard.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+    ![vdc_storage_nodes](./images/vdc_wallet.png)
+
+  - In the Backup And Restore tab there is a list of backups and the user can create new and restore any of them.
```
Comment (ranatrk) [inline docs/vdc/vdc_dashboard.md:70]:
```suggestion
  - `Backup And Restore tab`: consists of a list of backups. It also enables the user to create new ones or restore any of the previously backed up ones.
```

**Review by ranatrk — CHANGES_REQUESTED** [review]

**Comment by xmonader** [comment]
Let's also add some info on the included services in the package.

File: docs/vdc/vdc_dashboard.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+  │   └── api.js
+  │   └── index.html
+  │   └── app.vue
+  ├── package.py
+  ├── package.toml
+  └── __init__.py
+  ```
+
+### Background Services
+- `domain.py`: This service is responsible for redeploy subdomain.
```
Comment (ranatrk) [inline docs/vdc/vdc_dashboard.md:40]:
```suggestion
- `domain.py`: This service is responsible for redeploying subdomains.
```

File: docs/vdc/vdc_dashboard.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+- `etcd_backup.py`: This service is responsible for use `etcdctl` to get a snapshot of etcd and back it up using `restic`, *it stopped for now*
```
Comment (ranatrk) [inline docs/vdc/vdc_dashboard.md:41]:
```suggestion
- `etcd_backup.py`: This service is responsible for backup using `etcdctl` to get a snapshot of etcd and backing it up using `restic`. [currently disabled]
```

File: docs/vdc/vdc_dashboard.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+- `k8s_auto_extend.py`: This service is responsible for auto extend the kubernetes cluster to its plan limits if needed.
+- `provisioning_wallet_billing.py`: This service is responsible for extend the VDC pool.
+- `s3_auto_topup.py`: *it stopped for now*
+- `transaction_from_prepain_to_provisioning.py`: This service is responsible for transfer funds from prepaid to provision wallet
+- `zdb_auto_topup.py`: This service is responsible for extend the VDC ZDBs capacity.
```
Comment (ranatrk) [inline docs/vdc/vdc_dashboard.md:46]:
```suggestion
- `k8s_auto_extend.py`: This service is responsible for auto extending the kubernetes cluster to its plan limits if needed.
- `provisioning_wallet_billing.py`: This service is responsible for extending the VDC pool.
- `s3_auto_topup.py`: *it stopped for now*
- `transaction_from_prepain_to_provisioning.py`: This service is responsible for transfering funds from the prepaid to the provision wallet
- `zdb_auto_topup.py`: This service is responsible for extending the VDC ZDB's capacity.
```

**Review by ranatrk — APPROVED** [review]

**Review by xmonader — APPROVED** [review]

**Pull request merged**